src/utils.js

```diff
@@ -240,3 +240,57 @@ export const getSortedNotes = (resource, field, types) => {
 
   return sortedNotes;
 };
+
+/**
+ * Filter instance indetifiers by given type and
+ * return them as a comma separated string.
+ *
+ * @param indetifiers array of objects
+ * @param type string
+ * @param identifierTypesById object
+ *
+ * @return string
+ */
+export const getIdentifiers = (indetifiers = [], type, identifierTypesById) => {
```

Comment:
'identifiers' is mistyped as 'indetifiers' in several places

Reply:
Thank you for catching this @Baroquem!